### PR TITLE
[Feature-3862] Add Flink On Yarn Clusters e2e test

### LINKED ISSUE #3862: [Feature] Add Flink On Yarn Clusters e2e test
### Search before asking

- [X] I had searched in the [feature](https://github.com/apache/incubator-streampark/issues?q=is%3Aissue+label%3A%22Feature%22) and found no similar feature requirement.

### Description

including createCluster/startCluster/stopCluster/deleteCluster/editCluster.

### Usage Scenario

_No response_

### Related issues

_No response_

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by SbloodyS**
## What changes were proposed in this pull request

Issue Number: close #3862

## Brief change log

- Add Flink On Yarn Clusters e2e test
- Optimize some e2e code
- Fix add flink cluster ui bug

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

## Does this pull request potentially affect one of the following parts
 - Dependencies (does it add or upgrade a dependency): (yes / no)

### CODE REVIEW

File: .github/workflows/e2e.yml [hunk 1]
```diff
@@ -126,6 +126,12 @@ jobs:
             class: org.apache.streampark.e2e.cases.TeamManagementTest
           - name: MemberManagementTest
             class: org.apache.streampark.e2e.cases.MemberManagementTest
+          - name: Flink116OnYarnClustersTest
+            class: org.apache.streampark.e2e.cases.Flink116OnYarnClustersTest
```
Comment (wolfboys) [inline .github/workflows/e2e.yml]: Flink supports multiple clusters: yarn-session, standalone, and k8s-session. However, only yarn-session is currently implemented. Is the name of this test case ambiguous?

Comment (SbloodyS) [reply .github/workflows/e2e.yml]: Maybe using `Flink116OnYarnSessionClustersTest`? Do you have better idea?

Comment (wolfboys) [reply .github/workflows/e2e.yml]: This PR provides test cases for deploy and shutdown yarn-session cluster, How about Flink116OnYarnClusterDeployTest ?

Comment (SbloodyS) [reply .github/workflows/e2e.yml]: Okay. I'll fix it.

**Review by wolfboys — APPROVED** [review]
LGTM

**Pull request merged**